### DISCUSSION TITLE
Support for Browser based authentication popup

### DISCUSSION THREAD

**Discussion by asutoshbiswal1995** [cypress-io/cypress#17997] [General]
Hi Team,

Some URLs just show a popup at the launch for authentication. Which are mainly like the javascript based alerts/prompts.
There are no way that we can handle it using Cypress libraries as far as I know.

Could you please add browser based authentication handler in upcoming releases of Cypress?

**Comment by tracy-ash** [comment]
I would love support for popups generally.
